src/endpoints/Stories/Client.ts

```diff
@@ -166,95 +145,84 @@ export class Client {
         }
 
         if (isArray && typeof arg[0] === 'number') {
-            const { stories } = await this.search<Include, SearchOptions<Include>, StoryRecord>({
+            const { stories } = await this.search({
                 limit: arg.length,
                 query: { id: { $in: arg } },
-                include,
+                include: [...EXTENDED_STORY_INCLUDED_EXTRA_FIELDS, ...(include ?? [])],
```

Comment:
Btw, there was a bug here: the types declare it returns `ExtendedStory+Include`, but this (and the next) branch could return a narrower `Story+Include` interface